docs/contribution.html

```diff
@@ -93,25 +117,35 @@ <h2 id="methodes">Methodes</h2>
                     <li>private</li>
                 </ul>
             </li>
-            <li>Every methode should have a SINCE annotation (not needed if the class/struct/union already has one)</li>
+            <li>Every method should have a SINCE annotation (not needed if the class/struct/union already has one)</li>
```

Comment:
This needs to be removed for T4 on the Annotations are deprecated and will no longer be supported